src/selectors/exportContext.ts

```diff
@@ -15,6 +16,8 @@ const replaceTitle = (text: string, title: string, format: MimeType) => {
 
 /** Strips out HTML tags when exporting as plain text. */
 const stripHTMLTag = (str: string) => {
+  if (isMobile()) return str
+
```

Comment:
You need to strip out the HTML tags here. Can you see if this library works [`html-to-text`] in native (https://www.npmjs.com/package/html-to-text) ?

Reply:
Used regex instead.
`return str.replace(/(<([^>]+)>)/gi, '')`